.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: node_js
+sudo: false
 node_js:
-  - "7"
+  - "8"
```

Comment:
It is unclear why Node 7 no longer tested, as well as 5

Reply:
node 5 and 7 are no longer maintained by the Node.js Foundation (even-numbered versions are designated for long-term support, but odd-numbered versions are only supported 2 months after a new LTS release comes out. The full details are on the [nodejs blog post about the 8.0.0 release](https://nodejs.org/en/blog/release/v8.0.0/#long-term-support).

9 should probably be added though.

![node release schedule](https://talks.continuation.io/nodeweek-4-17/images/lts.png)

Reply:
Ah, right. Fair enough.